addons/sale/models/sale.py

```diff
@@ -289,6 +292,26 @@ def copy_data(self, default=None):
             default['order_line'] = [(0, 0, line.copy_data()[0]) for line in self.order_line.filtered(lambda l: not l.is_downpayment)]
         return super(SaleOrder, self).copy_data(default)
 
+    @api.multi
+    def name_get(self):
```

Comment:
Wait, what ? No. I want to see SO 003, not SO 003 Raoulette Poiluchette from Boobs Company.